sdk/messaging/azservicebus/messageBatch.go

```diff
@@ -111,12 +104,16 @@ func (mb *MessageBatch) Len() int {
 
 // toAMQPMessage converts this batch into a sendable *amqp.Message
 // NOTE: not idempotent!
-func (mb *MessageBatch) toAMQPMessage() (*amqp.Message, error) {
+func (mb *MessageBatch) ToAMQPMessage() *amqp.Message {
```

Comment:
Does this need to be exported?  Seems like an internal implementation detail.

Reply:
No, byproduct of a separate idea. Unexported.